client/package.json

```diff
@@ -97,17 +97,24 @@
     "@apollo/client": "3.0.2",
     "@blueprintjs/core": "3.30.0",
     "@blueprintjs/datetime": "3.18.4",
+    "@emotion/core": "^10.0.28",
+    "@emotion/styled": "^10.0.27",
```

Comment:
Pin version number (`yarn add -E @emotion/core @emotion/styled`) as that is what _renovate_ likes.